docs/antora.yml

```
@@ -1,12 +1,12 @@
 name: imdg
 title: Hazelcast IMDG
-version: 4.1.2
+version: 4.1.2-SNAPSHOT
 prerelease: true
 asciidoc:
   attributes:
     javasource: ROOT:example$
     page-toclevels: 1
-    page-ghissue: https://github.com/hazelcast/hazelcast-reference-manual/issues/new
+    page-ghissue: https://github.com/hazelcast/hazelcast-reference-manual/issues/new

```

```suggestion
    page-ghissue: https://github.com/hazelcast/imdg-docs/issues/new
```

This was pointing to the wrong repository.